addons/zylann.scatter/tools/plugin.gd

```diff
@@ -320,17 +335,17 @@ func _on_Palette_pattern_added(path):
 	# TODO Duh, may not work if the file was moved or renamed... I'm tired of this
 	var ur = get_undo_redo()
 	ur.create_action("Add scatter pattern")
-	ur.add_do_method(self, "add_pattern", path)
-	ur.add_undo_method(self, "remove_pattern", path)
+	ur.add_do_method(self, "_add_pattern", path)
```

Comment:
Your commit includes a fix from another PR, it was merged so you may want to update

Reply:
I'll give git a try :)